daemon/algod/api/algod.oas2.json

```diff
@@ -1693,18 +1693,48 @@
           "type": "string"
         },
         "sig-type": {
-          "description": "Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig",
+          "description": "Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig\n* or null if unknown",
```

Comment:
This seems to be documenting a change introduced by the indexer in https://github.com/algorand/indexer/pull/718 but is not relevant to algod